pyrokinetics/local_species.py

```diff
@@ -167,3 +167,18 @@ def from_kinetics(self,
 
         self['pressure'] = pressure
         self['a_lp'] = a_lp
+
+    def calculate_a_lp(self):
```

Comment:
This looks like a separate change?

Reply:
You think it's worth taking it out and leave the changes for #18 ?

Reply:
I don't mind really just flagging that this change looked like it wasn't directly a part of the main feature being introduced here.